downloader.php

```diff
@@ -1,41 +1,52 @@
 <?php
 $sticker_id = $_SERVER["argv"][1];
+
 if (!isset($sticker_id)) {
 	exit("no sticker id");
 } else if (preg_match("/^\d+$/", $sticker_id) != 1) {
 	exit("sticker id not a number");
 }
-$url  = "http://store.line.me/stickershop/product/".$sticker_id."/zh-Hant";
+
+$url = "http://store.line.me/stickershop/product/" . $sticker_id;
```

Comment:
Because I want to get Chinese folder name. I will use another parameter so that users can customize their language.